book.cmd/src/main/java/com/happystays/book/cmd/api/commands/PaymentInfo.java

```diff
@@ -0,0 +1,12 @@
+package com.happystays.book.cmd.api.commands;
+
+import lombok.Data;
+
+@Data
+public class PaymentInfo {
+    private String cardCode;
+    private String cardNumber;
+    private String seriescode;
+    private String expiryDate;
+    private String cardHolderName;
```

Comment:
Do we need a paymentType? can we ename cardcode as cardType and seriesCode as cvv